src/projects.h

```diff
@@ -226,12 +226,18 @@ typedef struct ARG_list {
 	/* base projection data structure */
 
 
+#ifdef PJ_LIB__
+    /* we need this forward declaration in order to be able to add a
+       pointer to struct opaque to the typedef struct PJconsts below */
+    struct opaque;
```

Comment:
Shouldn't struct opaque be prefixed with pj_ or PJ to avoid potential "namespace" clashes with code including proj headers ?